apps/native/app/src/screens/document-detail/document-detail.tsx

```diff
@@ -248,6 +249,17 @@ export const DocumentDetailScreen: NavigationFunctionComponent<{
   const [pdfUrl, setPdfUrl] = useState('')
   const [refetching, setRefetching] = useState(false)
 
+  const [logConfirmedAction] = useDocumentConfirmActionsLazyQuery({
+    fetchPolicy: 'no-cache',
+  })
+
+  const confirmAction = async (confirmed: boolean) => {
+    // Adding a suffix '_app' to the id since the backend is currently not distinguishing between the app and the web
+    await logConfirmedAction({
+      variables: { input: { id: `${docId}_app`, confirmed: confirmed } },
```

Comment:
Has the service been updated to support this suffix?

Reply:
No it just logs it, so I think it should be fine for now 😊 It expects a string